Ginger/Ginger/ApplicationModelsLib/ModelOptionalValue/AddOptionalValuesModelSelectTypePage.xaml.cs

```diff
@@ -171,6 +172,14 @@ private void BrowseFiles(bool MultiSelect)
             {
                 dlg.Filter = "Json files (*.json)|*.json|Text files (*.txt)|*.txt";
             }
+            else if (xSourceTypeComboBox.SelectedValue.ToString() == eSourceType.DB.ToString())
+            {
+                if (Convert.ToString(xDBTypeComboBox.SelectedValue) == eDBTypes.MSAccess.ToString())
+                {
+                    dlg.InitialDirectory = Path.Combine(((Amdocs.Ginger.Repository.RepositoryItemBase)WorkSpace.Instance.Solution).ContainingFolderFullPath, "DataSource");
```

Comment:
1- why we pointing as defualt to DataSource folder in case of MS Access? 
2- for getting the Solution folder you need to use "Worksapce.Instance.UserProfile.Solution.Folder"

Reply:
we pointing as defualt to DataSource folder in case of MS Access - since user might not be able to find the access file from solution folder so pointing to datasource folder.

- will change to use the "Worksapce.Instance.UserProfile.Solution.Folder"

Reply:
ok removed the initial directory assignment